<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QWidget" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3240</width>
    <height>2160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>3240</width>
    <height>2160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3240</width>
    <height>2160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainScreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: dark-gray;
color: white;
}
QWidget{
background-color: dark-gray;
color: white;
}
QLabel{
font: 21pt &quot;Helvetica Neue&quot;;
}
QLabel#labelLive{
font: 16pt &quot;Helvetica Neue&quot;;
font-weight: 600;
color: #262626;
}
QFrame{
border: 0px;
}
QPushButton{
font: bold 12pt;
qproperty-flat: true;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>530</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>80</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/octConsole/L300menuButton</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCondensUp">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/octConsole/condensUp</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDownArrow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/octConsole/dropdown</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCapture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../octConsole.qrc">
          <normaloff>:/octConsole/capture</normaloff>
          <normalon>:/octConsole/cameraYellow</normalon>:/octConsole/capture</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRecord">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../octConsole.qrc">
          <normaloff>:/octConsole/recordWhite</normaloff>
          <normalon>:/octConsole/recordRed</normalon>:/octConsole/recordWhite</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMeasure">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../octConsole.qrc">
          <normaloff>:/octConsole/measureWhite</normaloff>
          <normalon>:/octConsole/measureYellow</normalon>
          <disabledoff>:/octConsole/measureGrey</disabledoff>
          <disabledon>:/octConsole/measureGrey</disabledon>:/octConsole/measureWhite</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/octConsole/settingsW</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">color: &quot;black&quot;</string>
        </property>
        <property name="text">
         <string>SledOn</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEndCase">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: bold 36px &quot;Helvetica Neue&quot;;
border-radius: 75px;
background-color: rgb(72,13,2);</string>
          </property>
          <property name="text">
           <string>END
CASE</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameM">
     <property name="minimumSize">
      <size>
       <width>2159</width>
       <height>2159</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2159</width>
       <height>2159</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>2159</width>
          <height>2159</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>2159</width>
          <height>2159</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">&quot;border:5px solid rgb(0,0,0);&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameR">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>80</number>
      </property>
      <item>
       <widget class="QFrame" name="frameStatus">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>80</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <widget class="QLabel" name="labelCurrentTime">
             <property name="minimumSize">
              <size>
               <width>270</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelLive">
             <property name="text">
              <string>LIVE</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelRunTime">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Runtime: 0:00:00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelDevice">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameSpeed">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 4px solid grey; 
border-radius: 50px;
font: 20pt &quot;Helvetica Neue&quot;;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>120</number>
         </property>
         <property name="topMargin">
          <number>80</number>
         </property>
         <property name="rightMargin">
          <number>120</number>
         </property>
         <item>
          <widget class="QLabel" name="labelSpeed">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>SPEED</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLow">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>LOW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonMedium">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>MEDIUM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonHigh">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>HIGH</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>329</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelSim">
        <property name="text">
         <string>Frame </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFlip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/octConsole/flip</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../octConsole.qrc"/>
 </resources>
 <connections/>
</ui>
